content/reference-amendments.md

```diff
@@ -0,0 +1,365 @@
+
+Introduces three new transaction types: CheckCreate, CheckCancel, and CheckCash, and a new ledger object type, Check. Adds a new transaction result code, `tecEXPIRED`, which occurs when trying to create a Check whose expiration time is in the past.
+
+This amendment also changes the OfferCreate transaction to return `tecEXPIRED` when trying to create an Offer whose expiration time is in the past. Without this amendment, an OfferCreate whose expiration time is in the past returns `tesSUCCESS` but does not create or execute an Offer.
+
+**Caution:** This amendment is [in development](https://github.com/ripple/rippled/pull/2245) and is expected for `rippled` v0.90.0.
+
+## CryptoConditions
+[CryptoConditions]: #cryptoconditions
+
+| Amendment ID                                                     | Status  |
+|:-----------------------------------------------------------------|:--------|
+| 1562511F573A19AE9BD103B5D6B9E01B3B46805AEC5D3C4805C902B514399146 | Enabled |
+
+Although this amendment is enabled, it has no effect unless the [SusPay](#suspay) amendment is also enabled. Ripple does not expect SusPay to become enabled. Instead, Ripple plans to incorporate crypto-conditions in the [Escrow](#escrow) amendment.
+
+
+## CryptoConditionsSuite
+[CryptoConditionsSuite]: #cryptoconditionssuite
+
+| Amendment ID                                                     | Status  |
+|:-----------------------------------------------------------------|:--------|
+| 86E83A7D2ECE3AD5FA87AB2195AE015C950469ABF0B72EAACED318F74886AE90 | In Development |
+
+Implements several types of crypto-conditions from the official [crypto-conditions specification](https://tools.ietf.org/html/draft-thomas-crypto-conditions-03) for use in [EscrowCreate][] and [EscrowFinish][] transactions. Without this amendment, only the PREIMAGE-SHA-256 type is supported.
+
+**Caution:** This amendment is still [in development](https://github.com/ripple/rippled/pull/2170). The version from `rippled` v0.60.0 to present does not implement the full functionality.
+
+
+## DepositAuth
+[DepositAuth]: #depositauth
+
+| Amendment ID                                                     | Status  |
+|:-----------------------------------------------------------------|:--------|
+| F64E1EABBE79D55B3BB82020516CEC2C582A98A6BFE20FBE9BB6A0D233418064 | In Development |
+
+Adds a new account flag, `DepositAuth`, which lets an account strictly reject any incoming payments that are not signed by the receiver. This includes payments of XRP and of issued currencies. This also includes EscrowFinish and PaymentChannelClaim transactions not sent by the destination account. Businesses can use this flag to comply with strict regulations that require due diligence before receiving money from any source. As an exception, accounts with `DepositAuth` enabled can receive small amounts of XRP (equal or less than the minimum [account reserve](concept-reserves.html)) if their current XRP balance is below the account reserve.
```

Comment:
FWIW, the reason I picked the name `DepositAuth` is because a "payment" is already a transaction type and `DepositAuth` affects more transactions than just payments.  So I'm unsure about "...reject any incoming payments that...".  As Dave Cohen has pointed out, the term "Deposit" is not necessarily crystal clear either, unfortunately.  And it might be good to mention Checks in this context.  Consider something like the following introductory sentences:

"Adds a new account flag, `DepositAuth`, which only allows funds to be received by (deposited into) the account by transactions that are signed by the receiver.  This includes `Payments` and `Checks` (once enabled) of XRP and of issued currencies."

Other than those minor considerations, looks good.  As usual, you are the professional writer, not me.  These are only suggestions.